src/status_im/communities/core.cljs

```diff
@@ -784,3 +778,43 @@
                                 :community-id community-id
                                 :public-key   public-key
                                 :role-id      role-id})]})
+
+(rf/defn handle-fetched-collapsed-community-categories
+  {:events [:communities/fetched-collapsed-community-categories]}
+  [{:keys [db]} categories]
+  {:db
+   (assoc
```

Comment:
This code looks a bit funny, probably due to the code formatter :disappointed: A bit closer to how we would format and which still is kept intact by zprint is:

```clojure
(rf/defn handle-fetched-collapsed-community-categories
  {:events [:communities/fetched-collapsed-community-categories]}
  [{:keys [db]} categories]
  {:db (assoc db
              :communities/collapsed-categories
              (reduce
               (fn [acc {:keys [communityId categoryId]}]
                 (assoc-in acc [communityId categoryId] true))
               {}
               categories))})
```